File: solutions/965-unique-email-addresses/unique-email-addresses.cs
public class Solution {
    public int NumUniqueEmails(string[] emails) {
        var uniqueEmails = new HashSet<string>();
        
        foreach (string email in emails) {
            string[] parts = email.Split('@');
            string local = parts[0];
            string domain = parts[1];
            
            local = local.Split('+')[0];
            local = local.Replace(".", "");
            
            uniqueEmails.Add($"{local}@{domain}");
        }
        
        return uniqueEmails.Count;
    }
}

File: solutions/965-unique-email-addresses/unique-email-addresses.py
class Solution:
    def numUniqueEmails(self, emails: List[str]) -> int:
        ret = set()
        for email in emails:
            local, domain = email.split('@')
            local = local.split('+')[0]
            local = local.replace('.', '')
            ret.add((local, domain))
        return len(ret)